lib/amazon-pricing/definitions/database-type.rb

```diff
@@ -143,7 +231,61 @@ def self.display_name_is_byol?(display_name)
       database_name.include? 'byol'
     end
 
-  	def display_name
+    # example: database_sf?('MySQL Community Edition (Multi-AZ)') returns true
+    # Returns BOOL if database string is RDS SF
+    # params:
+    # - display_name[String]: fully qualified database string
+    def self.database_sf?(display_name)
+      db = @@DB_ENGINE_MAP[display_name]
+      return false unless db            # unknown db is presumed non sf
+      db[:sizeflex]
+    end
+    # example: operation_sf?('CreateDBInstance:0016',true) returns true
+    # Returns BOOL if operation string is RDS SF
+    # params:
+    # - operation_name[String]: fully qualified operation string
+    # - multiaz[bool]: if operation is multi-az (api for consistency purposes)
```

Comment:
If you don't decide to use multiaz. I think you'd better just remove it from the arguments.

Reply:
ironically, *you* are the one whom suggested that I add it:  specifically, it belongs in the API, but the (current) implementation doesn't use it.  your argument was that the client of the API shouldn't assume multiaz is _not_ a determinant whether it is SF.